Update Antlr version.

markww opened the pull request.

harsha-mandadi reviewed (commented):
Looks like the builds are failing.

markww commented:
Fixed it. The `rules_antlr` upstream is dead and hasn't been updated in 3 years. I forked it and added support for antlr 4.11.1 in my fork. I added a PR to roll it in, but it's unlikely it will be noticed, so we'll probably just use my fork from here on out. At least we know how to contact the maintainer, even if he knows almost nothing about the code.
